mozilla-release/browser/installer/windows/nsis/installer.nsi

```diff
@@ -992,13 +992,13 @@ Function preComponents
 
   ; Only show the maintenance service page if we have write access to HKLM
   ClearErrors
-  WriteRegStr HKLM "Software\CLIQZ" \
+  WriteRegStr HKLM "Software\Cliqz" \
```

Comment:
Nothing. Windows is case insensitive, even with registry. New users will receive new way of writing, for old users it must stay the same. I checked some cases - looks like it work as normal. But we will need to test this more with QA